holidays/countries/guyana.py

```diff
@@ -0,0 +1,279 @@
+        * [Extraordinary Gazette 26 April 2025](https://web.archive.org/web/20250611200310/https://officialgazette.gov.gy/images/gazette2025/apr/Extra_26APRIL2025PHA.pdf)
+    * Commonwealth Day / Emancipation Day:
+        * No amendment act renaming Commonwealth Day to Emancipation Day but being referred as
+          the latter in annual Gazette notifications for years.
+        * [Extraordinary Gazette 21 July 2016](https://web.archive.org/web/20250211095036/https://officialgazette.gov.gy/images/gazettes-files/Extraordinary-gazette_21jul16.pdf)
+        * [Extraordinary Gazette 16 July 2022](https://web.archive.org/web/20231214011437/https://officialgazette.gov.gy/images/gazette2022/jul/Extra_16JULY2022NotificPHA.pdf)
+    """
+
+    country = "GY"
+    default_language = "en_GY"
+    # %s (estimated).
+    estimated_label = tr("%s (estimated)")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observed, estimated)")
+    start_year = 1968
+    supported_languages = ("en_GY", "en_US")
+
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=GuyanaHinduHolidays)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=GuyanaIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        StaticHolidays.__init__(self, cls=GuyanaStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")), rule=SAT_SUN_TO_NEXT_MON)
+
+        if self._year <= 1969 or self._year >= 2016:
+            # Independence Day.
+            self._add_holiday_may_26(tr("Independence Day"))
+
+        if self._year >= 1970:
+            # Republic Day.
+            self._add_holiday_feb_23(tr("Republic Day"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Labour Day")))
+
+        if self._year >= 2019:
+            # Arrival Day.
+            self._add_observed(self._add_holiday_may_5(tr("Arrival Day")))
+
+        if self._year >= 2016:
+            # CARICOM Day.
+            self._add_holiday_1st_mon_of_jul(tr("CARICOM Day"))
+
+        self._add_holiday_1st_mon_of_aug(
+            # Commonwealth Day.
+            tr("Commonwealth Day")
+            if self._year <= 2015
+            # Emancipation Day
```

Comment:
```suggestion
            # Emancipation Day.
```
Last fix (here and in .po files).